nipype/interfaces/spm/model.py

```diff
@@ -151,7 +153,11 @@ def _parse_inputs(self):
         """validate spm realign options if set to None ignore
         """
         einputs = super(Level1Design,
-                        self)._parse_inputs(skip=('mask_threshold'))
+                        self)._parse_inputs(skip=('mask_threshold', 'flags'))
+        if isdefined(self.inputs.flags):
+            einputs[0].update(
+                {flag: val
+                 for (flag, val) in self.inputs.flags.items()})
```

Comment:
This could just be:

```Python
einputs[0].update(self.inputs.flags)
```

Reply:
same snippet is used elsewhere in the file, in EstimateModel class - we can perhaps leave it as it was tested and for consistency sake, or we change it at both places?